<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>generateMRS</class>
 <widget class="QDialog" name="generateMRS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate MRS</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>    MRS No</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Area/Stage</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLineEdit" name="lineEdit_MRS">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    border: 1px solid #a9a9a9;
    padding: 5px;
    border-radius: 10px; /* Adjust the value to control the roundness */
    background-color: #ffffff; /* Set the background color to white */
    color: #333333;
}

QLineEdit:focus {
    border-color: #3e7bb6;
}

QLineEdit:hover {
    background-color: #e6f3ff; /* Set the hover background color to light blue */
  
}

@keyframes blink {
    from {
        background-color: #e6f3ff;
    }
    to {
        background-color: #ffffff;
    }
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Project</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>   Date</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="lineEdit_Project">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    border: 1px solid #a9a9a9;
    padding: 5px;
    border-radius: 10px; /* Adjust the value to control the roundness */
    background-color: #ffffff; /* Set the background color to white */
    color: #333333;
}

QLineEdit:focus {
    border-color: #3e7bb6;
}

QLineEdit:hover {
    background-color: #e6f3ff; /* Set the hover background color to light blue */
  
}

@keyframes blink {
    from {
        background-color: #e6f3ff;
    }
    to {
        background-color: #ffffff;
    }
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Date">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    border: 1px solid #a9a9a9;
    padding: 5px;
    border-radius: 10px; /* Adjust the value to control the roundness */
    background-color: #ffffff; /* Set the background color to white */
    color: #333333;
}

QLineEdit:focus {
    border-color: #3e7bb6;
}

QLineEdit:hover {
    background-color: #e6f3ff; /* Set the hover background color to light blue */
   
}

@keyframes blink {
    from {
        background-color: #e6f3ff;
    }
    to {
        background-color: #ffffff;
    }
}
</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="styleSheet">
        <string notr="true">QTableView {
    border: 1px solid #d9d9d9;
    background-color: #ffffff;
    padding: 0;
    gridline-color: #d9d9d9;
}

QTableView::item {
    padding: 5px;
    border: none;
}

QTableView::item:selected {
    background-color: #3e7bb6;
    color: #ffffff;
}

QHeaderView::section {
    background-color: #f2f2f2;
    color: #333333;
    padding: 5px;
    font-weight: bold;
    border: none;
    border-right: 1px solid #d9d9d9;
    border-bottom: 1px solid #d9d9d9;
    min-width: 80px;
    min-height: 30px;
}

QHeaderView::section:checked {
    background-color: #3e7bb6;
    color: #ffffff;
}

QHeaderView::section:pressed {
    background-color: #3e7bb6;
    color: #ffffff;
}

QHeaderView::section:first {
    border-top-left-radius: 5px;
}

QHeaderView::section:last {
    border-top-right-radius: 5px;
    border-right: none;
}

QHeaderView::section:middle {
    border-right: none;
}

QTableView::horizontalHeader {
    border-top: 1px solid #d9d9d9;
    border-bottom: 1px solid #d9d9d9;
}

QTableView::verticalHeader {
    border-right: 1px solid #d9d9d9;
}

QHeaderView::section:horizontal {
    background-color: #e2e2e2;
    border-top: 1px solid #9d9d9d;
}

QHeaderView::section:horizontal:pressed {
    background-color: #3e7bb6;
    color: #ffffff;
}
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Part Number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Part_num">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border: 1px solid #a9a9a9;
    padding: 5px;
    border-radius: 10px; /* Adjust the value to control the roundness */
    background-color: #ffffff; /* Set the background color to white */
    color: #333333;
}

QLineEdit:focus {
    border-color: #3e7bb6;
}

QLineEdit:hover {
    background-color: #e6f3ff; /* Set the hover background color to light blue */

}

@keyframes blink {
    from {
        background-color: #e6f3ff;
    }
    to {
        background-color: #ffffff;
    }
}
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton{

	

	font: 14pt &quot;Times New Roman&quot;;  

		border-right: 1px solid #aaaaaa;

		border-bottom: 1px solid #aaaaaa;



}



QPushButton:enabled {

		background-color: green;

		color: white;

}



QPushButton:pressed {

		background-color: green;

		color: white;

}





QPushButton:hover:!pressed {

		background-color: ORANGE;

		color: red;

}



QPushButton:disabled {

		background-color: #aaaaaa;

		color: #ffffff;

}</string>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Manufacturer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_manf">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>QTY Available</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_qty_avail">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid #a9a9a9;
    padding: 5px;
    border-radius: 10px; /* Adjust the value to control the roundness */
    background-color: #ffffff; /* Set the background color to white */
    color: #333333;
}

QLineEdit:focus {
    border-color: #3e7bb6;
}

QLineEdit:hover {
    background-color: #e6f3ff; /* Set the hover background color to light blue */

}

@keyframes blink {
    from {
        background-color: #e6f3ff;
    }
    to {
        background-color: #ffffff;
    }
}
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Enter Qty</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_insert_in_inventory">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="asert.qrc">
           <normaloff>:/Images/RightArrow.ico</normaloff>:/Images/RightArrow.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_enter_qty">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid #a9a9a9;
    padding: 5px;
    border-radius: 10px; /* Adjust the value to control the roundness */
    background-color: #ffffff; /* Set the background color to white */
    color: #333333;
}

QLineEdit:focus {
    border-color: #3e7bb6;
}

QLineEdit:hover {
    background-color: #e6f3ff; /* Set the hover background color to light blue */

}

@keyframes blink {
    from {
        background-color: #e6f3ff;
    }
    to {
        background-color: #ffffff;
    }
}
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_add_qt_inventory">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="asert.qrc">
           <normaloff>:/Images/RightArrow.ico</normaloff>:/Images/RightArrow.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>288</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Authorised By</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Authorised">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border: 1px solid #a9a9a9;
    padding: 5px;
    border-radius: 10px; /* Adjust the value to control the roundness */
    background-color: #ffffff; /* Set the background color to white */
    color: #333333;
}

QLineEdit:focus {
    border-color: #3e7bb6;
}

QLineEdit:hover {
    background-color: #e6f3ff; /* Set the hover background color to light blue */

}

@keyframes blink {
    from {
        background-color: #e6f3ff;
    }
    to {
        background-color: #ffffff;
    }
}
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Select Recipient :-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_recepient_names">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>358</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton{

	

	font: 14pt &quot;Times New Roman&quot;;  

		border-right: 1px solid #aaaaaa;

		border-bottom: 1px solid #aaaaaa;



}



QPushButton:enabled {

		background-color: green;

		color: white;

}



QPushButton:pressed {

		background-color: green;

		color: white;

}





QPushButton:hover:!pressed {

		background-color: ORANGE;

		color: red;

}



QPushButton:disabled {

		background-color: #aaaaaa;

		color: #ffffff;

}</string>
         </property>
         <property name="text">
          <string>Generate Slip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ref">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton{

	

	font: 14pt &quot;Times New Roman&quot;;  

		border-right: 1px solid #aaaaaa;

		border-bottom: 1px solid #aaaaaa;



}



QPushButton:enabled {

		background-color: green;

		color: white;

}



QPushButton:pressed {

		background-color: green;

		color: white;

}





QPushButton:hover:!pressed {

		background-color: ORANGE;

		color: red;

}



QPushButton:disabled {

		background-color: #aaaaaa;

		color: #ffffff;

}</string>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="asert.qrc"/>
 </resources>
 <connections/>
</ui>
